concrete/single_pages/register.php

```diff
@@ -70,10 +69,12 @@
                         <?= $form->label('uPassword', t('Password')) ?>
                         <?= $form->password('uPassword', array('autocomplete' => 'off')) ?>
                     </div>
-                    <div class="form-group">
-                        <?= $form->label('uPasswordConfirm', t('Confirm Password')) ?>
-                        <?= $form->password('uPasswordConfirm', array('autocomplete' => 'off')) ?>
-                    </div>
+                    <?php if (Config::get('concrete.user.registration.display_confirm_password_field')): ?>
+                        <div class="form-group">
+                            <?= $form->label('uPasswordConfirm', t('Confirm Password')) ?>
+                            <?= $form->password('uPasswordConfirm', array('autocomplete' => 'off')) ?>
+                        </div>
+                    <?php endif; ?>
```

Comment:
Can you use full `if { }` syntax?

Reply:
https://github.com/concrete5/concrete5/pull/6021/commits/1e6614a092da87094980d4df4996fb0003b8a24f